pkg/metrics/latency_reporter.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
latency_reporter is too generic. Consider "informer_delay_reporter"

Reply:
Accounted for the name. It's now "brokercell_delay_reporter" -- more specific, but also let's us extend the reporter to other relevant delays within the brokercell.